qe-helper-scripts/create-vpc-infrastructure.sh

```
@@ -555,6 +561,184 @@ create_nat_gateway() {
+
+    aws events put-targets \
+        --rule "${QUEUE_PREFIX}-rebalance-recommendation" \
+        --targets "Id=1,Arn=${QUEUE_ARN}" \
+        --region "$REGION" > /dev/null
+
+    print_status "Setting queue policy to allow EventBridge to send messages..."
+    aws sqs set-queue-attributes \
+        --queue-url "$QUEUE_URL" \
+        --region "$REGION" \
+        --attributes '{
+    "Policy": "{\"Version\":\"2012-10-17\",\"Statement\":[{\"Effect\":\"Allow\",\"Principal\":{\"Service\":\"events.amazonaws.com\"},\"Action\":\"sqs:SendMessage\",\"Resource\":\"'${QUEUE_ARN}'\"}]}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)